pages/wizard/review/installation_review.py

```diff
@@ -0,0 +1,41 @@
+from selenium.webdriver.common.by import By
+from pages.base import Base
+
+
+class InstallationReview(Base):
+
+    # locators
+
+    _rhev_root_pw_eye_loc = (By.XPATH,
+                             '//a[@data-qci="root_password_engine_&_hypervisor"]/following-sibling::i')
+    _rhev_admin_pw_eye_loc = (By.XPATH,
+                              '//a[@data-qci="engine_admin_password"]/following-sibling::i')
+    _deploy_button_loc = (By.XPATH,
+                          '//button[text()="Deploy"]')
+
+    # elements
+
+    @property
+    def rhev_root_pw_eye_icon(self):
+        return self.selenium.find_element(*self._rhev_root_pw_eye_loc)
+
+    @property
+    def rhev_admin_pw_eye_icon(self):
+        return self.selenium.find_element(*self._rhev_admin_pw_eye_loc)
+
+    @property
+    def deploy_button(self):
+        return self.selenium.find_element(*self._deploy_button_loc)
+
+    # actions
+
+    def reveal_rhev_root_pw(self):
```

Comment:
Added reveal password for CFME admin, root, and database. There are others for OSP and OSE, but I'll need to do a deployment with those to get the locators for them to add in a separate PR.